pkg/bmc/bios.go

```diff
@@ -0,0 +1,149 @@
+package bmc
+
+import (
+	"reflect"
+	"strconv"
+
+	"github.com/gophercloud/gophercloud/openstack/baremetal/v1/nodes"
+
+	metal3v1alpha1 "github.com/metal3-io/baremetal-operator/pkg/apis/metal3/v1alpha1"
+)
+
+// A private method to build the clean steps for ILO configuration from BaremetalHost spec
+// TODO(asalkeld) I don't know which driver to associate this with
```

Comment:
I can't find a access.Driver() that returns "ilo", so I don't know where to put this